lib/services/livesync/ios-platform-livesync-service.ts

```diff
@@ -0,0 +1,64 @@
+import {PlatformLiveSyncServiceBase} from "./platform-livesync-service-base";
+
+class IOSPlatformLiveSyncService extends PlatformLiveSyncServiceBase {
+	constructor(_liveSyncData: ILiveSyncData,
+		protected $devicesService: Mobile.IDevicesService,
+		protected $mobileHelper: Mobile.IMobileHelper,
+		protected $logger: ILogger,
+		protected $options: ICommonOptions,
+		protected $deviceAppDataFactory: Mobile.IDeviceAppDataFactory,
+		protected $fs: IFileSystem,
+		protected $injector: IInjector,
+		protected $projectFilesManager: IProjectFilesManager,
+		protected $projectFilesProvider: IProjectFilesProvider,
+		protected $liveSyncProvider: ILiveSyncProvider) {
+		super(_liveSyncData, $devicesService, $mobileHelper, $logger, $options, $deviceAppDataFactory, $fs, $injector, $projectFilesManager, $projectFilesProvider, $liveSyncProvider);
+	}
+
+	public fullSync(): IFuture<void> {
+		return (() => {
+			let appIdentifier = this.liveSyncData.appIdentifier;
+			let platform = this.liveSyncData.platform;
+			let projectFilesPath = this.liveSyncData.projectFilesPath;
+			let canExecute = this.getCanExecuteAction(platform, appIdentifier);
+
+			let action = (device: Mobile.IDevice): IFuture<void> => {
+				return (() => {
+					let deviceAppData = this.$deviceAppDataFactory.create(appIdentifier, this.$mobileHelper.normalizePlatformName(platform), device);
+					let installed = this.tryInstallApplication(device, deviceAppData).wait();
+
+					if(installed) {
+						device.applicationManager.tryStartApplication(deviceAppData.appIdentifier).wait();
```

Comment:
We should add an option here to call different action (in case when running debug + livesync)